modules/core/output_modules.php

```diff
@@ -164,7 +164,13 @@ class Hm_Output_login_end extends Hm_Output_Module {
      * Closes the login form
      */
     protected function output() {
-        return '</form>';
+        $config = new Hm_Site_Config_File(CONFIG_FILE);
```

Comment:
Config is available in handler modules - pass whatever is needed to output modules and don't re-initialize the config class again here in an output module.

Reply:
Gave me headache, untill i remember we had a 'hello world' module which explained modules in cypht.
It is now implemented from handler module

Reply:
Yes, it is much better now, thanks!